.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: csharp
```

Comment:
Not a maintainer, but I feel like this would be better extracted as a separate PR, so it can be reviewed separately